sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -135,8 +134,8 @@
 ]
 
 
-def _hash_to_hex(hash):
-    return to_native_string(binascii.hexlify(hash))
+def _hash_to_hex(hash) -> str:
+    return binascii.hexlify(hash).decode("utf-8")
```

Comment:
`to_native_string` converts bytes `b'xxx'` to `"b'xxx'"`. The correct str representation of the hash should be properly decoded.